__tests__/costumes.test.js

```diff
@@ -0,0 +1,455 @@
+        `);
+
+        expect(chops.name).toBe("mutton chops");
+        expect(heels.name).toBe("platform heels");
+        expect(monocle.name).toBe("monocle");
+
+        const costumes = await getAllCostumes();
+        console.log(costumes)
+
+        expect(costumes).toContainEqual(chops);
+        expect(costumes).toContainEqual(heels);
+        expect(costumes).toContainEqual(monocle);
+    })
+})
+
+describe("getCostumeById adapter", () => {
+    it("should get costume that is first entry in table", async () => {
+        console.log("connected");
+        await createTables();
+        await createCostume(
+            "wool leggings",
+            "adult",
+            "female",
+            "M",
+            "tights",
+            3,
+            17.99
+        );
+        await createCostume(
+            "bloomers",
+            "baby",
+            "unisex",
+            "S",
+            "underwear",
+            5,
+            13.99
+        );
+        await createCostume(
+            "tank top",
+            "adult",
+            "unisex",
+            "L",
+            "shirt",
+            8,
+            18.99
+        );
+        const leggings = await getCostumeById(1);
+        expect(leggings.name).toBe("wool leggings");
+    })
+
+    it("should get costumes that is middle or last entry in table", async () => {
+        console.log("connected");
+        await createTables();
+        await createCostume(
+            "diamond grill",
+            "adult",
+            "unisex",
+            "M",
+            "jewelry",
+            1,
+            119.99
+        );
+        await createCostume(
+            "go-go boots",
+            "adult",
+            "female",
+            "XL",
+            "shoes",
+            2,
+            64.99
+        );
+        await createCostume(
+            "wayfarers",
+            "child",
+            "unisex",
+            "S",
+            "glasses",
+            1,
+            34.99
+        );
+        const wayfarers = await getCostumeById(3);
+        expect(wayfarers.name).toBe("wayfarers");
+        const boots = await getCostumeById(2);
+        expect(boots.name).toBe("go-go boots");        
+    })
+})
+
+//TODO: add test: update more than one costume in a row
+//TODO: add test: update the same costume more than once
+
+describe("updateCostume adapter", () => {
+    it("should update costume values when one value is changed", async () => {
+        console.log("connected");
+        await createTables();
+        await createCostume(
+            "short shorts",
+            "adult",
+            "unisex",
+            "M",
+            "pants",
+            2,
+            31.99
+        );
+        const shorts = await getCostumeById(1);
+        expect(shorts.name).toBe("short shorts");
+
+       await updateCostume(
+            1,
+            "very short shorts",
+            "adult",
+            "unisex",
+            "M",
+            "jewelry",
```

Comment:
seems odd

Reply:
Lol the shorts are so short that they are just a ring around your waist. You know, jewelry.